pkgs/development/interpreters/php/generic.nix

```diff
@@ -278,6 +285,10 @@ let
               substituteInPlace configure --replace "-lstdc++" "-lc++"
             '';
 
+          preInstall = ''
+            cp -f ${pearInstallPhar} ./pear/install-pear-nozlib.phar
```

Comment:
How is this change relevant?

Reply:
This is mandatory when building PHP from source. I haven't found a better way to do this, do you have a better idea?
When PHP release maintainers are doing a distribution, they run the script [`makedist`](https://github.com/php/php-src/blob/master/scripts/dev/makedist) that download that file using `wget`.